src/hooks/useApiBase.ts

```diff
@@ -439,7 +439,24 @@ const queryManifest = async (profileId: number) => {
             {
               org_id: true,
               role: true,
-              organization: { circles: [{}, { id: true }] },
+              organization: {
+                id: true,
+                name: true,
+                circles: [
+                  {},
```

Comment:
do we need to check for non-deleted circles? or the permissions should handle this?

Reply:
permissions should handle this